crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -46,29 +44,17 @@ def prepare
     end
   end
 
-  def services
```

Comment:
stopping services is not needed on cloud 6 (which was not that obvious from the commit message I admit)